app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        if self.health > 0:
+            Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return "{"\
+               + f"Name: {self.name}," \
+                 f" Health: {self.health}," \
+                 f" Hidden: {self.hidden}"\
+               + "}"
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
+        if self.hidden:
```

Comment:
![image](https://github.com/mate-academy/py-herbivores-and-carnivores/assets/104974881/10ed347b-28dd-486e-b353-b5e640fd32b3)